### DISCUSSION TITLE
Preventing screen saver and sleep mood?

### DISCUSSION THREAD

**Discussion by avishka-codimite** [react-native-tvos/react-native-tvos#399] [Feature Requests]
Hi, can I know if there is a way to prevent screen server and sleep mood on apple tv and android tv? I tried several libraries to achieve this task but none of these libraries worked for me. 

[react-native-keep-awake](https://github.com/corbt/react-native-keep-awake)
[@sayem314/react-native-keep-awake](https://github.com/sayem314/react-native-keep-awake)
[react-native-idle-timer](https://github.com/marcshilling/react-native-idle-timer)

**Comment by douglowder** [comment]
It's likely that these libraries are not built with the correct native support for controlling screen saver and sleep on TV platforms. I have not yet looked at these to see how support might be added.

I'll add this discussion to the "Feature Requests" section.

**Reply by avishka-codimite** [reply]
Also found this library. It's also not working with tvos. the library was quite old. 

[react-native-keep-awake-tvos](https://www.npmjs.com/package/react-native-keep-awake-tvos)
